caffe2/core/tensor.h

```diff
@@ -102,6 +102,25 @@ class CAFFE2_API Tensor final {
     return x;
   }
 
+  /**
+   * Clone self as a Tensor that share the same Storage
+   */
+  Tensor Alias() const {
```

Comment:
TODO: Deduplicate this with `THTensor_(newWithTensor)` (exposed in ATen as at::alias but not otherwise available)

```
/* Pointer-copy init */
THTensor *THTensor_(newWithTensor)(THTensor *tensor)
{
  THTensor *self = c10::make_intrusive<at::TensorImpl, at::UndefinedTensorImpl>(
    c10::intrusive_ptr<at::StorageImpl>::reclaim(THStorage_(new)()),
    at::CPUTensorId(),
    false
  ).release();
  THTensor_(setStorageNd)(self,
                          THTensor_getStoragePtr(tensor),
                          tensor->storage_offset(),
                          tensor->dim(),
                          THTensor_getSizePtr(tensor),
                          THTensor_getStridePtr(tensor));
  return self;
}
```

Reply:
you mean add this TODO in the comment right?

Reply:
probably better to do it in another diff?